packages/snowflake/queries/ddl/CREATE_DATABASE.SQL

```diff
@@ -0,0 +1,33 @@
+// Creating the initial Snowflake database to store ingested data from a Google Cloud Bucket
+
+USE ROLE SYSADMIN // or a role which is granted the appropriate permissions to create database, schema, and warehouse objects
+// TODO: create a role which is limited to only the exact permissions needed to execute this
+
+CREATE DATABASE NC_CAMPAIGN_FINANCE;
```

Comment:
If this is specific to the north carolina project we should probably move this as a sub-package of north-carolina